docs/src/styles/components/_menus.scss

```diff
@@ -61,18 +61,6 @@ html[data-theme="dark"] {
       padding-block-end: 0.5rem;
     }
 
-    .menu__list {
-      .menu__link--sublist {
-        margin-inline-start: calc(
-          -1 * (var(--dropdown-icon-width) + var(--dropdown-icon-gap))
-        );
-
-        &::after {
-          opacity: 0.6;
-        }
-      }
-    }
-
```

Comment:
I believe this was copied from what we had before docusaurus.  It messes up the indents of the new drop downs within drop downs.  It didn't apply to any nodes before this change.